src/api/fetchCityImage.jsx

```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+
+async function fetchCityImage(city) {
+  const response = await axios.get("https://api.unsplash.com/search/photos", {
+    headers: {
+      Authorization: `Client-ID ${process.env.REACT_APP_UNSPLASH_API_KEY}`,
+    },
+    params: { query: city },
+  });
+
+  return response.data.results[0].urls.small_s3;
+}
```

Comment:
Maybe something for later, but we should handle cases when we get an error from the API.

This can be through `.catch`, or a `try/catch` block. Then we can `alert` or just show some error message on screen.

But again, we can talk about this later on.

Reply:
Yes, good point. I've seen there are no images for some small cities already.